milton-manager/src/main/kotlin/io/terbium/milton/Milton.kt

```diff
@@ -120,6 +113,20 @@ class Milton @Inject constructor(
                         is PageManager.RegisterResult.Success -> call.respond(result.entry)
                     }
                 }
+                post("/delete") {
+                    if (call.authentication.principal == null) {
+                        return@post call.respond(HttpStatusCode.Forbidden, "authentication required")
+                    }
```

Comment:
#15 implemented the authentication functionality you're leveraging here, but it seems to me that to allow deletes you additionally need "authorization", meaning a list of users allowed to actually delete things. (Logging which user deleted a given url also seems prudent, and would add "accounting/ auditing", and give you the final A in AAA)

Otherwise this functionality doesn't protect against an authenticated troll deleting all our articles.